unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/UnmodifiableMutableListTest.java

```diff
@@ -205,6 +206,18 @@ public void removeFromIndex()
         Verify.assertThrows(UnsupportedOperationException.class, () -> this.unmodifiableList.remove(0));
     }
 
+    @Test
+    public void replaceAll()
+    {
+        Verify.assertThrows(UnsupportedOperationException.class, () -> this.unmodifiableList.replaceAll(e -> e));
```

Comment:
I prefer having Verify.assertThrows() but if rest of the tests are @ throws, then these ones should also be consistent.

Reply:
Not sure what you want me to do here.  Is this inconsistent in this class or with other classes?

Reply:
Other classes should also use assertThrows

Reply:
Please feel free to fix those other classes.